tests/test_browser.py

```diff
@@ -3372,10 +3408,9 @@ def test_pthread_cancel(self):
     self.btest(path_from_root('tests', 'pthread', 'test_pthread_cancel.cpp'), expected='1', args=['-O3', '-s', 'USE_PTHREADS=1', '-s', 'PTHREAD_POOL_SIZE=8'])
 
   # Test pthread_kill() operation
+  # This test hangs the chrome render process, and keep subsequent tests from passing too
```

Comment:
can/should this comment be folded in to the no_chrome note? Seems almost redundant to have both